### PR TITLE
Default help key on authors

### PR DESCRIPTION
**Pull request by yroffin**
Like the repository key

### CODE REVIEW

File: webui/src/app/components/appdetail/appdetail.component.html [hunk 1]
```diff
@@ -50,7 +50,7 @@ <h5 class="oui-header_5">
         <hr>
 
         <h5 class="oui-header_5">
-          <i class="oui-icon oui-icon-user_line" aria-hidden="true"></i>
+          <app-help-widget key="help-what-is-a-contributor"></app-help-widget><i class="oui-icon oui-icon-user_line" aria-hidden="true"></i>
```
Comment (rayene) [inline webui/src/app/components/appdetail/appdetail.component.html]: The help icon should appear after the "Authors" string.

File: samples/mycompany.sh [hunk 2]
```diff
@@ -27,6 +27,17 @@ An application is a piece of software that
...
+  --url $API_BASE_URL/v1/contents/help-what-is-a-contributor/en-GB \
+  --header 'content-type: text/plain' \
+  --data 'Authors
+------
+
+A contributor is someone who commit on code repository.
+
+A maintainer is someone who actively support this application.
+'
+
```
Comment (rayene) [inline samples/mycompany.sh]: I do not agree fully on the definition of a maintainer. We can keep it simple here and put a fake value to close the discussion.

Comment (yroffin) [reply samples/mycompany.sh]: What is your definition of a maintainer, just give it to me ... and i update the PR ?

File: webui/src/app/components/appdetail/appdetail.component.html (hunk 1 above)
Comment (yroffin) [reply webui/src/app/components/appdetail/appdetail.component.html]: Done

**Comment by ovh-cds** [comment]
CDS Report deploy-preview#267.0

* 
  * deploy-platform ✔

**Pull request merged**